maintainers/maintainer-list.nix

```diff
@@ -8161,6 +8161,11 @@
     githubId = 16481032;
     name = "Kiba Fox";
   };
+  kidanger = {
+    github = "kidanger";
+    githubId = 297479;
+    name = "Jérémy Anger";
```

Comment:
It wouldn't hurt to add another mean of contact

Reply:
I've added the email information.